src/main/scala/scalismo/statisticalmodel/StatisticalMeshModel.scala

```diff
@@ -215,8 +216,12 @@ object StatisticalMeshModel {
   }
 
   /**
-   *  Adds a bias model to the given statistical shape model
+   *  @deprecated
```

Comment:
It would be helpful to add a short message, which the users sees during compilation
```@deprecated("some reason","20-04-2016")```